### PR TITLE
Add multiple hostzones to LocalNet

### PR DESCRIPTION
**Pull request by vish-stride**
**Summary**

Adds multiple host zones to local net (specifically Juno)

**Test plan**

`make init-local build=sghij; make test-integration` should pass

### CODE REVIEW

File: scripts-local/tests/integration_tests.bats [hunk 1]
```diff
@@ -44,6 +42,11 @@ setup() {
 ######                              SETUP TESTS                                         ######
 ##############################################################################################
 
+@test "test" {
+  assert_equal "HI" "HI"
```
[the remark this answers is not in the document]
Comment (riley-stride) [reply scripts-local/tests/integration_tests.bats]: did you figure this out? happy to help, no use spinning on bats issues

[the remark this answers is not in the document]
Comment (vish-stride) [reply scripts-local/tests/integration_tests.bats]: actually if you have a few minutes, would be really helpful to get your take! [](edited-with-Axolo)

**Review by asalzmann — APPROVED** [review]
lgtm - did you resolve the bats issues?

**Comment by riley-stride** [comment]
What's the status of this PR? Seems to be superseded by more recent PRs

**Pull request closed without merging**